src/main/java/org/commcare/formplayer/annotations/AppInstall.java

```diff
@@ -1,4 +1,4 @@
-package annotations;
+package org.commcare.formplayer.annotations;
```

Comment:
This is going to be the most annoying nitpick in the world, but shouldn't this be `org.commcarehq.formplayer`? I always thought you were supposed to use a namespace for which you own the DNS entry. We don't own commcare.org (right?) but we do own commcarehq.org

Reply:
`org.commcare` is the root namespace for [commcare core](https://github.com/dimagi/commcare-core/tree/master/src/main/java/org/commcare) and for [commcare android](https://github.com/dimagi/commcare-android/tree/master/app/src/org/commcare) (since they were all in use before there was a www.commcarehq.org) so it made the most sense to me to maintain that consistency. It's occasionally relevant for filtering code in profiling contexts or in packaging contexts, so it would add a meaningful effort to use both `org.commcare` and `org.commcarehq`

Owning the DNS entry is more of a "It just feels weird not to" best practice. If we were starting fresh today I'd definitely use commcarehq, but with the historical technical weight it's not really worth it.

Reply:
Ah okay gotcha that all makes sense